apps/site/package.json

```diff
@@ -66,7 +67,7 @@
     "mime-types": "^2.1.35",
     "mock-block-dock": "0.0.39",
     "mongodb": "^4.2.2",
-    "next": "13.0.7",
+    "next": "13.1.0",
```

Comment:
`next@13.1.6` is out, which may contain bugfixes in `transpilePackages`. I’d suggest upgrading, unless you see reasons to stay on `13.1.0`.